### DISCUSSION TITLE
Welcome to F1-api Discussions!

### DISCUSSION THREAD

**Discussion by Rafacv23** [Rafacv23/F1-api#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PabloBlanco10** [comment]
Hello @Rafacv23, it would be awesome to have a yaml file with all the endpoints in order to be able to use openapi generator, what do you think? thank you in advance
